styles/audio.css

```diff
@@ -38,6 +38,19 @@
   content: "\f028";
 }
 
+.h5p-audio-inner .h5p-audio-minimal-play-paused, .h5p-audio-inner .h5p-audio-minimal-pause {
+  background: rgb(38,170,68); /* Old browsers */
+  background: -moz-linear-gradient(top,  rgba(38,170,68,1) 0%, rgba(0,112,10,1) 100%); /* FF3.6+ */
+  background: -webkit-gradient(linear, left top, left bottom, color-stop(0%,rgba(38,170,68,1)), color-stop(100%,rgba(0,112,10,1))); /* Chrome,Safari4+ */
+  background: -webkit-linear-gradient(top,  rgba(38,170,68,1) 0%,rgba(0,112,10,1) 100%); /* Chrome10+,Safari5.1+ */
+  background: -o-linear-gradient(top,  rgba(38,170,68,1) 0%,rgba(0,112,10,1) 100%); /* Opera 11.10+ */
+  background: -ms-linear-gradient(top,  rgba(38,170,68,1) 0%,rgba(0,112,10,1) 100%); /* IE10+ */
+  filter: progid:DXImageTransform.Microsoft.gradient( startColorstr='#6498fe', endColorstr='#0468ce',GradientType=0 ); /* IE6-9 */
```

Comment:
No need to support IE6-9...

Reply:
Just used it because the other currently used selector has it, too ;-)